tracer/src/Datadog.Trace.Tools.Runner/RunCiCommand.cs

```diff
@@ -308,6 +314,48 @@ public override async Task<int> ExecuteAsync(CommandContext context, RunCiSettin
                             session.SetTag(CommonTags.CodeCoverageTotalLines, globalCoverage.Data[0].ToString(CultureInfo.InvariantCulture));
                         }
                     }
+                    else
+                    {
+                        try
+                        {
+                            if (EnvironmentHelpers.GetEnvironmentVariable(Configuration.ConfigurationKeys.CIVisibility.ExternalCodeCoveragePath) is { Length: > 0 } extCodeCoverageFilePath &&
```

Comment:
If both `DD_CIVISIBILITY_CODE_COVERAGE_ENABLED` and `DD_CIVISIBILITY_EXTERNAL_CODE_COVERAGE_PATH` are set, can you add a warning saying something like:
> DD_CIVISIBILITY_EXTERNAL_CODE_COVERAGE_PATH will be ignored because DD_CIVISIBILITY_CODE_COVERAGE_ENABLED is set.

Reply:
done in [b4d570f](https://github.com/DataDog/dd-trace-dotnet/pull/4069/commits/b4d570f8c2571811a2114e616ca553d063b2d8d0)